Sources/Purchasing/Offerings.swift

```diff
@@ -59,7 +68,7 @@ import Foundation
 
     private let currentOfferingID: String?
     private let placements: Placements?
-    private let targeting: Targeting?
+    internal let targeting: Targeting?
```

Comment:
Changing to internal so the custom attribute keys can be used in `CustomAttributesManager`